src/test/java/org/openmrs/contrib/qaframework/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -0,0 +1,463 @@
+	private static final By CURRENT_DATE = By.linkText("Today");
+	private static final By VISIT_NOTE_ENCOUNTER = By.xpath("//div[@id='visit-details']/ul/li/ul/li/div/strong/span[text()='Visit Note']");
+	private static final By SAVE_VISIT_NOTE = By.cssSelector(".submitButton.confirm");
+	private static final By CONFIRM_DEL_BUTTON = By.cssSelector("#delete-encounter-dialog > div.dialog-content > button.confirm.right");
+	private static final By DEL_DIAGNOSIS = By.xpath("//div[@id='display-encounter-diagnoses-container']/ul/li/span/i");
+	private static final By DATE_FIELD = By.id("w5-display");
+	private static final By PROVIDER = By.id("w1");
+	private static final By LOCATION = By.id("w3");
+	private static final By WHO_WHEN_WHERE = By.id("who-when-where");
+	private static final By REQUEST_APPOINTMENT = By.linkText("Request Appointment");
+	private static final By APPOINTMENT_LINK = By.cssSelector("a.right > i");
+	private static final By ERROR = By.cssSelector("li.error > span");
+	private static final By ADD_PAST_VISIT = By.id("org.openmrs.module.coreapps.createRetrospectiveVisit");
+	private static final By TODAY = By.cssSelector("td.day.active");
+	private static final By CONFIRM_PAST_VISIT = By.cssSelector("div.dialog-content.form > button.confirm.right");
+	private static final By CHANGE_DATE = By.cssSelector("button.confirm.no-color");
+	private static final By DAY = By.xpath("//table[@class=' table-condensed']/tbody/tr[1]/td[0]");
+	private static final By MERGE_VISIT_BUTTON = By.xpath("//a[@id='org.openmrs.module.coreapps.mergeVisits']/li");
+	private static final By MERGE = By.id("mergeVisitsBtn");
+	private static final By PATIENT = By.xpath("//ul[@id='breadcrumbs']/li[2]/a");
+	private static final By TELEPHONE_NUMBER_TEXT = By.cssSelector("#coreapps-telephoneNumber");
+	private static final By RECENT_VISITS = By.cssSelector("visitbyencountertype > ul > li:nth-child(1) > a");
+	private static final By PATIENT_GIVENNAME = By.cssSelector("#content div span.PersonName-givenName");
+	private static final By ACTIVE_VISIT_MESSAGE = By.cssSelector("active-visit-message");
+	private static final By ALLERGIES_LINK = By.id("allergyui-editAllergies");
+	private static final By CONDITIONS_LINK = By.cssSelector(".conditions .info-header i.right");
+	private static final By ATTACHMENTS_LINK = By.cssSelector("a[id='attachments.attachments.visitActions.default']");
+	private static final By ACTIVE_VISIT = By.cssSelector("visitbyencountertype ul li a");
+	private static final By DELETE_PATIENT_LINK = By.id("org.openmrs.module.coreapps.deletePatient");
+	private static final By DELETE_PATIENT_REASON = By.id("delete-reason");
+	private static final By DELETE_PATIENT_CONFIRM = By.cssSelector("#delete-patient-creation-dialog > div.dialog-content > button.confirm.right");
+	private static final By DELETE_PATIENT_CANCEL = By.cssSelector("#delete-patient-creation-dialog > div.dialog-content > button.cancel");
```

Comment:
I do not see where DELETE_PATIENT_CANCEL is used. Do you really need it?

Reply:
These functionalities are used in deletePatientTests

Reply:
Is there any function that complains when you comment  DELETE_PATIENT_CANCEL?